eva/parser/evaql/evaql_parser.g4

```diff
@@ -154,10 +154,16 @@ updateStatement
     : singleUpdateStatement
     ;
 
+
 loadStatement
     : LOAD DATA
       INFILE fileName
       INTO tableName
+      (WITH fileOptions)?
+    ;
+
+fileOptions
+    : FORMAT fileFormat=(CSV|VIDEO)
```

Comment:
It seems that the design is that file format is one possible file options. Then we may only need file options across node, plan, and executor instead of file format and file options.

Reply:
Got it got it. I will make this change thanks